ResearchKit/Common/ORKAnswerFormat.h

```diff
@@ -84,6 +85,17 @@ typedef NS_ENUM(NSInteger, ORKChoiceAnswerStyle) {
     ORKChoiceAnswerStyleMultipleChoice
 } ORK_ENUM_AVAILABLE;
 
+/// An enumeration of the format styles available for scale answers.
+typedef NS_ENUM(NSInteger, ORKNumberFormattingStyle) {
+    /// The default decimal style.
+    ORKNumberFormattingStyleDefault,
+    
+    /// Percent style.
+    ORKNumberFormattingStylePercent,
+    
+    /// Currency style.
+    ORKNumberFormattingStyleCurrency
```

Comment:
Hmm, the result would then need to report what currency was used. Otherwise you have no record of the units of measure.